helm/tests/testdata/command_args.yaml

```
@@ -0,0 +1,4 @@
+coder:
+  image:
+    tag: latest
+  commandArgs: []
```

```suggestion
  commandArgs:
    - arg1
    - arg2
```

And regenerate with the same command as before. We want to test that setting this value to something works, by leaving it as `[]` the test doesn't really do anything because that's the default value.